packages/core/echo/echo-schema/src/text.ts

```diff
@@ -6,23 +6,28 @@ import get from 'lodash.get';
 
 import { next as A } from '@dxos/automerge/automerge';
 
-import { createDocAccessor } from './automerge';
+import { type DocAccessor, createDocAccessor, type KeyPath } from './automerge';
 import { type EchoReactiveObject } from './effect/reactive';
 
 /**
  * @deprecated
  */
 export const LEGACY_TEXT_TYPE = 'dxos.Text.v0';
 
-// TODO(burdon): Reconcile with cursorConverter.
-
-// TODO(wittjosiah): Path shouldn't be hardcoded.
-const path = ['content'];
-
-export const toCursor = <T>(object: EchoReactiveObject<T>, pos: number) => {
-  const accessor = createDocAccessor(object, path);
-  const doc = accessor.handle.docSync();
-  if (!doc) {
+export const toCursor = <T>({
```

Comment:
hmm, why not `toCursor(createDocAccessor(obj, path), index)` to make the API leaner?

Reply:
e.g. having only one way of doing something...